sdk/core/azure-core-amqp/test/ut/amqp_message_tests.cpp

```diff
@@ -133,7 +133,7 @@ TEST_F(TestMessage, TestBodyAmqpSequence)
   {
     AmqpMessage message;
 
-    message.SetBody({"Test", 95, static_cast<AmqpValue>(AmqpMap{{3, 5}, {4, 9}})});
+    message.SetBody({"Test", 95, AmqpMap{{3, 5}, {4, 9}}.AsAmqpValue()});
```

Comment:
BTW, do you want to also (in the future) remove `operator X` from the other AMQP types, and replace them with `AsX()`?

Reply:
No I do not. The AsAmqpValue overload exists explicitly to enable conversion of AMQP collection types back to an AmqpValue - for some reason, the compiler didn't want to invoke the operator AmqpValue() overload.